docs/subnets/deploying-subnets-on-prod.md

```diff
@@ -0,0 +1,187 @@
+
+Before running the wizard you will need to [create the `WMID`](subnet-cli.md#subnet-cli-create-vmid), have the validator NodeIDs ready, as well as the genesis `json` file.
+
+An example of wizard command line could look like:
+
+```bash
+subnet-cli wizard \
+--ledger \
+--node-ids=NodeID-741aqvs6R4iuHDyd1qT1NrFTmsgu78dc4,NodeID-K7Y79oAmBntAcdkyY1CLxCim8QuqcZbBp,NodeID-C3EY6u4v7DDi6YEbYf1wmXdvkEFXYuXNW,NodeID-AiLGeqQfh9gZY3Y8wLMD15tuJtsJHq5Qi \
+--vm-genesis-path=prod-genesis.json \
+--vm-id=tGas3T58KzdjLHhBDMnH2TvrddhqTji5iZAMZ3RXs2NLpSnhH \
+--chain-name=prodSubnet
+```
+
+Please refer to the Subnet-CLI docs for [detailed instructions](subnet-cli.md#subnet-cli-wizard).
+
+If the wizard completes correctly your Subnet will now be created, deployed and available to see in [the Subnet explorer](https://subnets.avax.network/subnets). Congrats!
+
+You can also use Subnet-CLI to manage your Subnet after the initial deploy, to add new validators, renew ones that expired and so on. Familiarize yourself with Subnet-CLI, you be using it often!
+
+## Validator Configuration
+
+Running nodes as Subnet validators warrants some additional considerations, above those when running a regular node or a Primary Network-only validator.
+
+### Subnet Whitelisting
+
+For a node to join a subnet, there are two prerequisites:
+* Primary Network validation
+* Subnet whitelisting
+
+Primary Network validation means that a node cannot join a subnet as a validator before becoming a validator on the Primary Network itself. So, after you add the node to the validator set on the Primary Network, node can join a subnet. Of course, this is valid only for subnet validators, if you need a non-validating subnet node, then the node doesn't need to be a validator at all.
+
+To have a node start syncing the Subnet, you need to add the `--whitelisted-subnets` command line option, or `whitelisted-subnets` key to the node config file (found at `.avalanchego/configs/node.json` for installer-script created nodes). A single node can sync multiple subnets, so you can add them as a comma-separated list of subnet IDs.
+
+An example of a node config syncing two subnets:
+
+```json
+{
+  "dynamic-public-ip": "opendns",
+  "http-host": "",
+  "whitelisted-subnets": "28nrH5T2BMvNrWecFcV3mfccjs6axM1TVyqe79MCv2Mhs8kxiY,Ai42MkKqk8yjXFCpoHXw7rdTWSHiKEMqh5h8gbxwjgkCUfkrk"
+}
+```
+
+But that is not all. Besides the whitelist containing the SubnetID, node also needs to have the plugin that contains the VM instance the blockchain in the Subnet will run. You should have already been through that on testnet and Fuji, but for a refresher, you can refer to [this tutorial](create-a-fuji-subnet.md).
+
+So, name the VM plugin binary as the `VMID` of the Subnet chain and place it in the `plugins` directory where the node binary is (for installer-script created nodes that would be `~/avalanche-node/plugins/`). 
```

Comment:
It seems a little strange to reference this here. I guess this is more of a "considerations for deploying on production" document than a walk through tutorial, but if we want this to be a walkthrough or have a section that does the full walkthrough, I think we should explicitly state in that walkthrough to make sure the needed VM binaries are in place.

Reply:
To make this an explicit walkthrough we would need to duplicate a lot of material from other tutorials (installation, subnet-cli and so on) and we wanted to avoid that (tools are rapidly evolving and copies get outdated, maintenance is messy). Instead we opted for linking to source material as that will be up to date.

Explicit mention of the plugin binary comes from experience, lots of people stumble on that detail, but I'm ok with removing it if you feel strongly about it.